akka-docs/_sphinx/themes/akka/static/warnOldDocs.js

```diff
@@ -0,0 +1,70 @@
+jQuery(document).ready(function ($) {
+
+  var latestInSeries = {
+    "2.2": {
+      latest: "2.2.5",
+      deprecated: true,
+      instead: "2.3"
+    },
+    "2.3": {
+      latest: "2.3.13"
+    },
+    "2.4": {
+      latest: "2.4.0-RC1"
+    },
+
+    // -- RP --
+    "rp-15v01": {
+      rp: true,
+      latest: "rp-15v01p05"
+    }
+  };
+
+  //if (location.hostname === "doc.akka.io") {
+  for (var series in latestInSeries) {
+    if (browsedVersion().startsWith(series)) {
+      floatyWarning(browsedVersion(), latestInSeries[series])
+    }
+  }
+  //}
+
+  function floatyWarning(version, seriesInfo) {
+    var targetUrl = ("" + window.location).replace(version, seriesInfo.latest);
+    var $floatyWarning = $('<div id="floaty-warning"/>');
+
+    $floatyWarning
+        .html('' +
+        'You are browsing the docs for Akka [' + version + '], ' +
```

Comment:
thanks for the feedback, good idea.
It's somewhat muscle memory, I do that with log statements :wink: